api/src/main/java/org/pmiops/workbench/api/EgressEventsAdminController.java

```diff
@@ -162,6 +155,24 @@
   @Override
   public ResponseEntity<AuditEgressEventResponse> auditEgressEvent(
       String id, AuditEgressEventRequest request) {
-    throw new NotImplementedException();
+    DbEgressEvent dbEgressEvent = mustGetDbEvent(id);
+
+    return ResponseEntity.ok(
+        new AuditEgressEventResponse()
+            .egressEvent(egressEventMapper.toApiEvent(dbEgressEvent))
+            .sumologicEvent(new Gson().fromJson(dbEgressEvent.getSumologicEvent(), Object.class))
```

Comment:
how sumologicEvent looks like in UI? Is that well formatted json string or just a long string without formatting?

Reply:
Preview screenshot here: https://github.com/all-of-us/workbench/pull/5979

Yes - we assume it is at least formatted JSON. However, we make no assumption of what keys it will contain.